src/main/java/xbony2/huesodewiki/prefix/CorrectTitlePrefix.java

```diff
@@ -5,18 +5,14 @@
 import xbony2.huesodewiki.api.IPagePrefix;
 
 public class CorrectTitlePrefix implements IPagePrefix {
-	/**
-	 * As defined at <a href="https://en.wikipedia.org/wiki/Wikipedia:Naming_conventions_(technical_restrictions)#Forbidden_characters">Wikipedia:Naming conventions (technical restrictions)#Forbidden characters</a>.
-	 */
-	private static final char[] INVALID_CHARS = { '#', '<', '>', '[', ']', '|', '{', '}' };
-
+	
 	@Override
 	public boolean canAdd(ItemStack itemstack){
-		return StringUtils.containsAny(itemstack.getDisplayName(), INVALID_CHARS);
+		return StringUtils.containsAny(itemstack.getDisplayName(), {'#', '<', '>', '[', ']', '|', '{', '}'}); // see: https://en.wikipedia.org/wiki/Wikipedia:NCHASHTAG
 	}
 
 	@Override
 	public String getText(ItemStack itemstack){
-		return String.format("{{Correct title|%s}}", itemstack.getDisplayName());
+		return "{{Correct title|", itemstack.getDisplayName() + "}}");
```

Comment:
This won't compile. It has a `,` instead of a `+` and a stranded `)`.

Reply:
that is problematic